setup.py

```diff
@@ -11,8 +11,9 @@
 ['jsonschema>=4.4.0,<5.0.0',
  'numpy>=1.21.6,<2.0.0',
  'requests>=2.27.1,<3.0.0',
- 'torch>=1.11.0',
- 'torchaudio>=0.11.0',
+ 'soundfile>=0.12.1',
+ 'torch>=1.11.0,<2.0.0',
+ 'torchaudio>=0.11.0,<2.0.0',
```

Comment:
Is there a way we can omit the torch version that runs really slowly after compiling? was it 1.12?

Reply:
Unfortunately we can't, it was all of them after 1.11. Removing the freeze from jit makes them all fast though, maybe I should add a warning about that there

Reply:
Feel free to do that in this PR or I can add it to the version bump one